code/game/gamemodes/cult/runes.dm

```diff
@@ -208,13 +209,24 @@
 						log_admin("[M]([ckey(M.key)]) ghosted/disconnected less than a minute after having been converted to the cult! ([T.x],[T.y],[T.z] - <A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[T.x];Y=[T.y];Z=[T.z]'>JMP</a>)")
 			return 1
 		else
-			if(jobban_isbanned(M, "cultist"))
-				M.Sleeping(300)//putting them to sleep for 5 minutes.
+			if(is_convertable_to_cult(M.mind) && jobban_isbanned(M, "cultist")) //Not implanted, but cultbanned
+				var/turf/T = get_turf(M)
+				T.turf_animation('icons/effects/effects.dmi',"rune_teleport")
+				M.unequip_everything() //Piñata
+				//death(M) //toggles SPS from going off or not.
+				sleep(1) //Ensure everything has time to drop without getting deleted
+				qdel(M)
```

Comment:
Discussed a bunch in the discord, general consensus is deletion + animation is the best course of action.